src/Setup/Setup.vcxproj

```diff
@@ -29,7 +29,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v120</PlatformToolset>
+    <PlatformToolset>v120_xp</PlatformToolset>
```

Comment:
It's an XML file so I expect the usual syntax works. However, it's typically generated by changes made using VS; the next one would probably trash my comment.